src/client/entity-editor/shared-data/shared-data.js

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (C) 2016  Ben Ockmore
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+import {Col, Row} from 'react-bootstrap';
+import AliasButton from './alias-button-container';
+import DisambiguationButton from './disambiguation-button-container';
+import DisambiguationField from './disambiguation-field-container';
+import IdentifierButton from './identifier-button';
+import LanguageField from './language-field-container';
+import NameField from './name-field-container';
+import React from 'react';
+import SortNameField from './sort-name-field-container';
+
+function SharedData({
+	disambiguationVisible,
+	languageOptions
+}) {
+	return (
+		<div>
+			<h2>
+				What is the Creator called?
```

Comment:
Mention of creators in a generic form. Also, how do you feel about adding little hint popups with this sort of text next to a basic "Name" field?